kahenya-anita/Bumble-Clone
# Hamida ft homepage

## Description
Created the home page HTML following BEM 
## Types Of Changes
[] Bug fix

[x] New Feature

[ ] Breaking Changes

## Checklist:
[x]My code follows the code style of this project

[x]All new and existing tests passed.

[x]My code has been linted and formatted correctly.

[]I have commented my code, particularly in hard-to-understand areas.

[x]My changes generate no new warnings.

## How can this code be tested?
1. After Cloning this repo , run `git pull
` to ensure you are up to date. 
2. Checkout to Branch
3. Run the changes in the browser
